### ISSUE TITLE
Storage api buckets no longer works (supabase 1.50.9)

### ISSUE THREAD

**Issue by thomydu57** [supabase/storage-api#315]
I installed the latest version of supabase (1.50.9), and since then the buckets no longer work. When I want to create one on the supabase interface, the error "new row violates row-level security policy" is displayed.

**Comment by fenos** [comment]
@thomydu57 Hey thanks for reporting this issue, is this happening in supabase local development?

If yes, could you try running this SQL to unblock the issue until we release a patch:

```
grant anon, authenticated, service_role to supabase_storage_admin;
```

**Comment by sweatybridge** [comment]
This will be addressed in CLI 1.50.12

**Comment by thomydu57** [comment]
It works, nice ! Thanks for this tips
